Makefile

```diff
@@ -34,6 +34,9 @@ cargo-sort:
 
 check: check-fmt check-clippy cargo-sort
 
+unit-test:
```

Comment:
Are these ran in the CI?

Reply:
The ci will run all tests, using the `make test` command. This command just makes running unit test convenient.